sdk/storage/azblob/recordings/Test/TestAccountDeleteRetentionPolicy-variables.yaml

```diff
@@ -0,0 +1,4 @@
+---
```

Comment:
Just for future notice, we're changing recording infrastructure and we'll have to migrate storage at some point

Reply:
Thanks for the information. Will keep that in mind.